huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_member_test.go

```diff
@@ -129,9 +129,10 @@ data "huaweicloud_vpc_subnet" "test" {
 data "huaweicloud_availability_zones" "test" {}
 
 resource "huaweicloud_elb_loadbalancer" "test" {
-  name            = "%s"
-  ipv4_subnet_id  = data.huaweicloud_vpc_subnet.test.subnet_id
-  ipv6_network_id = data.huaweicloud_vpc_subnet.test.id
+  name              = "%s"
+  cross_vpc_backend = true
```

Comment:
suggest add another acceptance test for cross_vpc_backend

Reply:
done